modules/gapi/src/backends/fluid/gfluidimgproc.cpp

```diff
@@ -1895,7 +1896,8 @@ static inline void initScratchLinear(const cv::GMatDesc& in,
     auto *clone = scr.clone;
     auto *index = scr.mapsx;
 
-    for (int x = 0; x < outSz.width; x++) {
+    for (int x = 0; x < outSz.width; ++x)
```

Comment:
from my perspective modern c++ compiler is smart enough in this automatization

Reply:
I'm not used to relying too much on compiler optimizations :-)